test/firefox/interfaces/getters.test.js

```diff
@@ -0,0 +1,13 @@
+import { getTelemetryID } from "../../../src/firefox/interfaces/getters";
+
+describe("firefox/interfaces/getters.js", () => {
+  describe("getTelemetryID()", () => {
+    it("should return a telemetry ID", async () => {
+      browser.experiments.firefox_bridge.getTelemetryID.mockResolvedValue(
```

Comment:
Can the tests actually make calls to browser.experiments.firefox_bridge? If so, you could write a test to validate that it's getting the value from the prefs, and you could even have it fall back to the default.

Services.prefs.getStringPref.mockResolvedValue("");
Services.prefs.getStringPref.mockResolvedValue("sample_telemetry_id");

expect(Services.prefs.getStringPref).toHaveBeenCalledWith("browser.firefoxbridge.installId", "");

Reply:
If this can be done, if can be done in a follow up patch, after the change to return "" instead of undefined.

Reply:
This would have to involve integration testing because I believe the privileged API only populates browser.experiments when the extension is loaded with the proper prefs. I'm not so knowledgeable in this area and documentation is sparse compared to the general API.

There are a bunch of things that need to be tested through integration tests that aren't set up yet.